apps/examples/nettest/nettest_stress.c

```diff
@@ -0,0 +1,376 @@
+#include <errno.h>
+#include <semaphore.h>
+#include <pthread.h>
+#include <arpa/inet.h>
+
+/*
+ *  * HEAPINFO
+ *   */
+#define PRINT_HEAPINFO 1
+
+#if PRINT_HEAPINFO
+#include <tinyara/mm/mm.h>
+static int disconnect_count = 0;
+#endif
+
+/*
+ *  * NETWORK
+ *   */
+#define AF_INETX AF_INET
+#define PF_INETX PF_INET
+#define PKT_NUM 10
+#define BUF_SIZE 301
+#define SELECT_TIME_OUT	100000
+
+/*
+ *  * WiFi
+ *   */
+#define WIFI_AUTO_CONNECT 1
+
+#if WIFI_AUTO_CONNECT
+#include <wifi_manager/wifi_manager.h>
+
+#define WIFI_SSID "Gorani"
+#define WIFI_PASSPHRASE "jonbeo1@"
+static sem_t g_wifi_sem = SEM_INITIALIZER(0);
+
+#define WIFITEST_SIGNAL                                          \
+	do {                                                         \
+		sem_post(&g_wifi_sem);                      \
+		printf("%s: T%d send signal\n", __FUNCTION__, getpid()); \
+	} while (0)
+
+#define WIFITEST_WAIT                                            \
+	do {                                                         \
+		printf("%s: T%d wait signal\n", __FUNCTION__, getpid()); \
+		sem_wait(&g_wifi_sem);                      \
+	} while (0)
+
+void wifi_sta_connected(wifi_manager_result_e result)
+{
+	WIFITEST_SIGNAL;
+}
+
+void wifi_sta_disconnected(wifi_manager_disconnect_e disconn)
+{
+	WIFITEST_SIGNAL;
+}
+
+void wifi_scan_ap_done(wifi_manager_scan_info_s **scan_info, wifi_manager_scan_result_e res)
+{
+	/* Make sure you copy the scan results onto a local data structure.
+	 *	 * It will be deleted soon eventually as you exit this function.
+	 *		 */
+	if (res == WIFI_SCAN_FAIL) {
+		WIFITEST_SIGNAL;
+		return;
+	}
+	wifi_manager_scan_info_s *wifi_scan_iter = *scan_info;
+	while (wifi_scan_iter != NULL) {
+		printf("SSID: %-20s, BSSID: %-20s, RSSI: %d, CH: %d, Phy_type: %d\n", \
+				wifi_scan_iter->ssid, wifi_scan_iter->bssid, wifi_scan_iter->rssi, \
+				wifi_scan_iter->channel, wifi_scan_iter->phy_mode);
+		wifi_scan_iter = wifi_scan_iter->next;
+	}
+	WIFITEST_SIGNAL;
+}
+static wifi_manager_cb_s wifi_callbacks = {
+	wifi_sta_connected,
+	wifi_sta_disconnected,
+	NULL,
+	NULL,
+	wifi_scan_ap_done,
+};
+#endif
+
+static int g_sockfd;
+static struct sockaddr_in g_myaddr;
+int terminate_flag = 0;
+int sending_now = 0;
+sem_t g_sem_signal;
+
+int retry_sender = 0;
+int retry_receiver = 0;
+
+void *sender_thread(void *arg)
+{
+	int ret = 0;
+	int i = 0;
+	uint32_t sbuf_size = 0;
+	char buf[BUF_SIZE];
+	memset(buf, 0xff, BUF_SIZE);
+	sbuf_size = BUF_SIZE;
+
+
+	/* Then send PKT_NUM number of messages */
+	while (1) {
+		printf("[SEND] Start to send\n");
+		for (i = 0; i < PKT_NUM; i++) {
+			sending_now = 1;
+			int send_sum = 0;
+			while (send_sum < BUF_SIZE) {
+				ret = send(g_sockfd, buf + send_sum, sbuf_size - send_sum, 0);
+				if (ret <= 0) {
+					if (ret == 0) {
+						printf("[SEND] connection closed\n");
+						goto terminate;
+					}
+					printf("[SEND] socket error ret(%d) err(%d)\n", ret, errno);
+					if (errno == 104 || errno == 9) {
```

Comment:
How about using defined value, like `ECONNRESET` instead of 104?

Reply:
Modified. Thanks for the comment.